Homework13/Tests/WebTableTests.cs

```diff
@@ -0,0 +1,139 @@
+﻿using Homework13.Data.Constants;
+using Homework13.Helpers;
+using Homework13.PageObjects;
+using NUnit.Framework;
+
+namespace Homework13.Tests
+{
+    public class WebTableTests : BaseTest
+    {
+        private string _firstName = RandomHelper.GetRandomString(10);
+        private string _lastName = RandomHelper.GetRandomString(10);
+        private string _email = $"{RandomHelper.GetRandomString(10)}@example.com";
+        private string _age = RandomHelper.GetRandomInt(100);
+        private string _salary = RandomHelper.GetRandomInt(100000);
+        private string _department = RandomHelper.GetRandomString(10);
+
+        [SetUp]
+        public void SetUp()
+        {
+            GenericPages.BaseElementsPage.ClickOnElementsMenuItem(Elements.WebTables);
+        }
+
+        // Test1: Check validation of form inputs.
+        [Test]
+        public void CheckValidation()
+        {
+            GenericPages.WebTablePage.ClickAddNewEntryButton();
+            GenericPages.WebTablePage.ClickSubmitButton();
+            Assert.IsTrue(GenericPages.WebTablePage.IsFormValidated());
+
+            // Postcondition: return to original state.
+            GenericPages.WebTablePage.ClickCloseButton();
+        }
+
+        // Test2: Search an entry.
+        [Test]
+        public void SearchEntry()
```

Comment:
There are not enough asserts to confirm search is working. You need to confirm that before search there were more rows than after

Reply:
Agree, if there was one row initially, then preconditions steps would be required)